celer/tests/test_lasso.py

```diff
@@ -218,6 +218,27 @@ def test_weights_lasso():
     np.testing.assert_raises(ValueError, clf1.fit, X=X, y=y)
 
 
+def test_infinite_weights():
+    np.random.seed(1)
+    n_samples, n_features = 50, 100
+    X = np.random.randn(n_samples, n_features)
+    y = np.random.randn(n_samples)
+
+    weights = np.ones(n_features)
+    weights[0] = np.inf
```

Comment:
put more weights to inf in order to have less chance of the corresponding coefs being 0